docs/modules/ROOT/pages/Introduction.adoc

```diff
@@ -72,9 +72,11 @@ To verify your pwa is compliant to the provided Dockerfile, you can use the Dock
 You just need to change the template file in such way, that your custom env parameters are set accordingly to you `.env` file.
 
 Once you have updated your template just rename the file to `Dockerfile`.
-If you then execute `docker build .` and the image could be built successfully, 
+If you then execute `docker build -t my-pwa .` and the image could be built successfully, 
 your application will be compliant with the e-Spirit build pipeline.
+To ensure the created image runs successfully, you can also run it locally with `docker run -p XXXX:3000 my-pwa`. 
+Where XXXX can be any free port. After the container is running, you can open localhost:XXXX in a browser and the the pwa should show up.
```

Comment:
I would change `After the container is running` to `When the container is running` and `open` to `visit`.

There is a double `the` at the end of the sentence.